Client/Frontend/Strings.swift

```diff
@@ -2445,6 +2445,12 @@ extension String {
             value: "Request Mobile Site",
             comment: "Label for the button, displayed in the menu, used to request the mobile version of the current website.",
             lastUpdated: .unknown)
+		public static let AppMenuReloadWithoutCacheTitleString = MZLocalizedString(
+			"Menu.RefreshWithoutCache.Title",
+			tableName: "Menu",
+			value: "Reload Ignoring Cache",
+			comment: "Label for the button, displayed in the menu, used to reload without cache.",
+			lastUpdated: .unknown)
```

Comment:
We can keep this string to use in the discoverabilityTitle of the keyboard shortcut but we'll need to change a couple of things. We should add this string under `KeyboardShortcuts` struct in the strings file, and not under `AppMenu`. You'll also need to add the v108 version in StringLastUpdatedAppVersion of that file to add the proper last updated param.
```suggestion
		public static let ReloadWithoutCache = MZLocalizedString(
			"Keyboard.Shortcuts.RefreshWithoutCache.v108",
			value: "Reload Ignoring Cache",
			comment: "A label indicating the keyboard shortcut to reload a tab without it's cache. This label is displayed in the Discoverability overlay when a user presses the Command key. The Discoverability overlay and shortcut become available only when a user has connected a hardware keyboard to an iPad.",
			lastUpdated: .v108)
```